frontend/packages/fonts/README.md

```diff
@@ -0,0 +1,54 @@
+# @code-dot-org/fonts
+
+This package contains the fonts used by Code.org.
+
+## Table of Contents
+
+- [Overview](#overview)
```

Comment:
This isn't linking anywhere, can it be removed or does this section need to be added?